doc/api/util.md

```diff
@@ -1792,6 +1792,31 @@ console.log(util.stripVTControlCharacters('\u001B[4mvalue\u001B[0m'));
 // Prints "value"
 ```
 
+## `util.styleText(format, text)`
+
+> Stability: 1.1 - Active development
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `format` {string} A text format defined in `util.inspect.colors`.
+* `text` {string} The text to to be formatted.
+
+This function returns a formatted text considering the `format` passed.
+
+```js
+const textRedCollr = util.styleText('red', 'My red message');
+console.log(textRedColor);
+```
```

Comment:
GitHub is going to butcher if I try to write this as a suggestion, but should this be `mjs` and `cjs` blocks with `import { styleText } from 'node:util';` and `const { styleText } = require('node:util');` at the top?

Reply:
> GitHub is going to butcher if I try to write this as a suggestion

Nope, you can totally do that, you just need to make it valid markdown, see:

````suggestion
```cjs
const textRedCollr = util.styleText('red', 'My red message');
console.log(textRedColor);
```
````

In this case, I don't think it's worth having two different version, I think it's fine to have just that one.

Reply:
> Nope, you can totally do that, you just need to make it valid markdown, see:

How do you do it to show both blocks?

I don’t like just having `util` as this looks-like-a-global where the docs assume that the user knows to add `import util from 'node:util'` or `const util = require('node:util')` above it. We don’t do that elsewhere in our docs that I’m aware of, like all the `fs` docs show the import/require lines in those examples.

Reply:
~~~~suggestion
```mjs
import { styleText } from 'node:util';

const errorMessage = styleText('red', 'Error! Error!');
console.log(errorMessage);
```

```cjs
const { styleText } = require('node:util');

const errorMessage = styleText('red', 'Error! Error!');
console.log(errorMessage);
```
~~~~